File: VideoOperations/src/main/java/com/simform/videooperations/LogMessage.java
package com.simform.videooperations;

import com.arthenica.mobileffmpeg.Level;

public class LogMessage {

    private final long executionId;
    private final Level level;
    private final String text;

    public LogMessage(final long executionId, final Level level, final String text) {
        this.executionId = executionId;
        this.level = level;
        this.text = text;
    }

    public long getExecutionId() {
        return executionId;
    }

    public Level getLevel() {
        return level;
    }

    public String getText() {
        return text;
    }

    @Override
    public String toString() {
        final StringBuilder stringBuilder = new StringBuilder();

        stringBuilder.append("LogMessage{");
        stringBuilder.append("executionId=");
        stringBuilder.append(executionId);
        stringBuilder.append(", level=");
        stringBuilder.append(level);
        stringBuilder.append(", text=");
        stringBuilder.append("\'");
        stringBuilder.append(text);
        stringBuilder.append('\'');
        stringBuilder.append('}');

        return stringBuilder.toString();
    }
}

File: VideoOperations/src/main/java/com/simform/videooperations/LogMessage.kt
package com.simform.videooperations

import com.arthenica.mobileffmpeg.Level

class LogMessage(val executionId: Long, val level: Level, val text: String) {
    override fun toString(): String {
        val stringBuilder = StringBuilder()
        stringBuilder.append("LogMessage{")
        stringBuilder.append("executionId=")
        stringBuilder.append(executionId)
        stringBuilder.append(", level=")
        stringBuilder.append(level)
        stringBuilder.append(", text=")
        stringBuilder.append("\'")
        stringBuilder.append(text)
        stringBuilder.append('\'')
        stringBuilder.append('}')
        return stringBuilder.toString()
    }
}